recipes/h5cpp/all/conanfile.py

```diff
@@ -0,0 +1,96 @@
+import os
+
+from conans import ConanFile, CMake, tools
+
+
+class H5cppConan(ConanFile):
+    name = "h5cpp"
+    version = "0.4.1"
+    license = "LGPL-2.1"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/ess-dmsc/h5cpp"
+    description = "h5cpp is a new C++ wrapper for HDF5s C-API"
+    topics = ("HDF5",)
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake"
```

Comment:
This is also missing

```suggestion
    generators = "cmake", "cmake_find_package"
```